tests/e2e/specs/plugin-updater.test.js

```diff
@@ -34,7 +34,7 @@ describe( 'Jetpack updater', () => {
 
 	it( 'Plugin updater', async () => {
 		await step( 'Can login and navigate to Plugins page', async () => {
-			await loginToWpSite();
+			// await loginToWpSite();
```

Comment:
Is this line intentionally being left here? Since this is test code, I don't have a preference for whether the line is commented out or removed. I just want to make sure that it's intentional.

Reply:
Good catch. it should be removed!